_printf.c

```diff
@@ -0,0 +1,6 @@
+#include "holberton.h"
+
```

Comment:
description missed, add description of your function

Reply:
Done